src/main/resources/templates/AdvisorRHEL/insightsEmailBody.html

```diff
@@ -580,7 +580,7 @@ <h1><b>{#insert content-title}{/}</b></h1>
                                 <tbody>
                                 <tr>
                                     <td class="rh-footer__text">
-                                        This email was sent by Red Hat Insights |<a href="https://cloud.redhat.com/user-preferences/notification/rhel" target="_blank">&nbsp;Manage notification preferences</a>
+                                        This email was sent by Red Hat Insights |<a href="https://cloud.redhat.com/user-preferences/notifications/openshift" target="_blank">&nbsp;Manage email preferences</a>
```

Comment:
The URL was correct here should be https://cloud.redhat.com/user-preferences/notifications/rhel I think ?
The text of the link was correct https://marvelapp.com/prototype/5gdjb12/screen/76689545